static/app/views/starfish/views/appStartup/averageComparisonChart.tsx

```diff
@@ -0,0 +1,127 @@
+import {useMemo} from 'react';
+
+import {t} from 'sentry/locale';
+import type {NewQuery} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import type {TableDataRow} from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {formatVersion} from 'sentry/utils/formatters';
+import {decodeScalar} from 'sentry/utils/queryString';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {useReleaseSelection} from 'sentry/views/starfish/queries/useReleases';
+import {SpanMetricsField} from 'sentry/views/starfish/types';
+import {formatVersionAndCenterTruncate} from 'sentry/views/starfish/utils/centerTruncate';
+import {appendReleaseFilters} from 'sentry/views/starfish/utils/releaseComparison';
+import {MAX_CHART_RELEASE_CHARS} from 'sentry/views/starfish/views/appStartup';
+import {COLD_START_TYPE} from 'sentry/views/starfish/views/appStartup/screenSummary/startTypeSelector';
+import {YAxis, YAXIS_COLUMNS} from 'sentry/views/starfish/views/screens';
+import {ScreensBarChart} from 'sentry/views/starfish/views/screens/screenBarChart';
+import {useTableQuery} from 'sentry/views/starfish/views/screens/screensTable';
+
+interface Props {
+  chartHeight?: number;
+}
+
+function transformData(data: TableDataRow[] | undefined, appStartType: string) {
+  if (!defined(data)) {
+    return [];
+  }
+
+  return [
+    {
+      seriesName: t('Average Duration'),
+      data:
+        data?.map(row => ({
+          name: formatVersion(row.release as string),
+          value:
+            (row[
+              YAXIS_COLUMNS[
+                appStartType === COLD_START_TYPE ? YAxis.COLD_START : YAxis.WARM_START
+              ]
+            ] as number) ?? 0,
+        })) ?? [],
+    },
+  ];
+}
+
+export function AverageComparisonChart({chartHeight}: Props) {
+  const location = useLocation();
+  const {
+    primaryRelease,
+    secondaryRelease,
+    isLoading: isReleasesLoading,
+  } = useReleaseSelection();
+  const pageFilter = usePageFilters();
+  const {selection} = pageFilter;
```

Comment:
Nit, looks like your never using pageFilter, and just the selection directly, this can be similified down here (and in other spots)
```suggestion
  const {selection} = usePageFilters();;
```